examples/IBFE/explicit/ex5/example.cpp

```diff
@@ -529,17 +536,15 @@ run_example(int argc, char** argv)
         // velocity.
         if (SAMRAI_MPI::getRank() == 0)
         {
-            drag_stream.open("C_D.curve", ios_base::out | ios_base::trunc);
-            lift_stream.open("C_L.curve", ios_base::out | ios_base::trunc);
-            U_L1_norm_stream.open("U_L1.curve", ios_base::out | ios_base::trunc);
-            U_L2_norm_stream.open("U_L2.curve", ios_base::out | ios_base::trunc);
-            U_max_norm_stream.open("U_max.curve", ios_base::out | ios_base::trunc);
-
-            drag_stream.precision(10);
-            lift_stream.precision(10);
-            U_L1_norm_stream.precision(10);
-            U_L2_norm_stream.precision(10);
-            U_max_norm_stream.precision(10);
+            drag_F_stream.open("C_F_D.curve", ios_base::out | ios_base::trunc);
+            lift_F_stream.open("C_F_L.curve", ios_base::out | ios_base::trunc);
+            drag_TAU_stream.open("C_T_D.curve", ios_base::out | ios_base::trunc);
```

Comment:
These only should be active if IIM is being used.

Reply:
okay I think this is the answer to my previous question.